narwhals/expr_str.py

```diff
@@ -447,3 +447,40 @@ def to_lowercase(self) -> ExprT:
         return self._expr._with_elementwise_op(
             lambda plx: self._expr._to_compliant_expr(plx).str.to_lowercase()
         )
+
+    def zfill(self, width: int) -> ExprT:
+        """Transform string to zero-padded variant.
+
+        Warning:
+            Different backends handle strings that start with "+" and "-" signs differently:
```

Comment:
Can this be removed? Or altered?

Reply:
Behavior should be consistent across all backends now, isn't it?

Reply:
done